src/components/input-text/input-text.tsx

```diff
@@ -50,10 +51,11 @@ const Label = styled.label<LabelProps>`
 export const InputText = ({
   id = "input-text",
   label,
+  value: inputValue = "",
   onChange,
   ...rest
 }: InputProps) => {
-  const [value, setValue] = useState("");
+  const [value, setValue] = useState(inputValue);
```

Comment:
What logic you need this state variable for?

Reply:
Yes we don;t need internal state. Removed it